src/controllers/product.controller.js

```diff
@@ -32,7 +32,39 @@ export const getAllProduct = async (req, res) => {
     });
   }
 };
+export const getAvailableProducts = async (req, res) => {
+  try {
+    const products = await models.Product.findAll({
+      where: { available: true }
+    });
+    res.json(products);
```

Comment:
inside json function add something like `res.json({response:products})` it will be easy for API consumers to know what they are getting as a result while doing their integration.